packages/twitch-pubsub-client/src/SingleUserPubSubClient.ts

```diff
@@ -146,6 +148,29 @@ export class SingleUserPubSubClient {
 		return this._addListener('chat_moderator_actions', callback, 'channel:moderate', extractUserId(channelId));
 	}
 
+	/**
+	 * Adds a listener to undocumented events to the client.
+	 *
+	 * @param topic The topic to subscribe to
+	 * @param callback A function to be called when a mod action event is sent to the user.
+	 * @param scope The scope required for the event subscription
+	 * @param channelId The ID of the channel to listen to, if the topic requires it.
+	 *
+	 * It receives a {@PubSubChatModActionMessage} object.
+	 */
+	async onUndocumentedAction(
```

Comment:
Also `onCustomTopic` - the names in this and `PubSubListener` should stay in sync.